algorithms/linfa-pls/src/pls_generic.rs

```diff
@@ -259,7 +264,7 @@ impl<F: Float, D: Data<Elem = F>> Fit<'_, ArrayBase<D, Ix2>, ArrayBase<D, Ix2>>
                 Algorithm::Nipals => {
                     // Replace columns that are all close to zero with zeros
                     for mut yj in yk.gencolumns_mut() {
-                        if *(yj.mapv(|y| num_traits::float::Float::abs(y)).max().unwrap())
+                        if *(yj.mapv(|y| num_traits::float::Float::abs(y)).max()?)
```

Comment:
does `y.abs()` now work?

Reply:
```
multiple applicable items in scope

multiple `abs` foundrustc(E0034)
```

Reply:
done in https://github.com/Sauro98/linfa/pull/3/commits/e9bdd75e66a6cd1a73cc52e132e96c2e64116228